client/src/assets/scss/main.scss

```diff
@@ -175,6 +175,12 @@ textarea {
   font-size: 14px;
 }
 
+.not-found {
```

Comment:
Just after noticing this added in here. Given it only applies to PageNotFound.vue, it should probably just be put in there.